store/sqlstore/user_store.go

```diff
@@ -1102,14 +1102,18 @@ func (us SqlUserStore) AnalyticsActiveCount(timePeriod int64, options model.User
 
 	time := model.GetMillis() - timePeriod
 
-	query := "SELECT COUNT(*) FROM Status s"
-
-	if options.IncludeBotAccounts {
-		query += " WHERE LastActivityAt > :Time"
-	} else {
-		query += " LEFT JOIN Bots ON s.UserId = Bots.UserId WHERE Bots.UserId IS NULL AND LastActivityAt > :Time"
+	joins := []string{}
+	wheres := []string{"LastActivityAt > :Time"}
+	if !options.IncludeBotAccounts {
+		joins = append(joins, "LEFT JOIN Bots ON s.UserId = Bots.UserId")
+		wheres = append(wheres, "Bots.UserId IS NULL")
 	}
 
+	if !options.IncludeDeleted {
+		joins = append(joins, "LEFT JOIN Users ON s.UserId = Users.Id")
+		wheres = append(wheres, "Users.DeleteAt = 0")
+	}
+	query := fmt.Sprintf("SELECT COUNT(*) FROM Status s %v WHERE %v", strings.Join(joins, " "), strings.Join(wheres, " AND "))
```

Comment:
I think in cases like these we want to use the query builder.

Reply:
Usually I agree, but my attempts of re-implementing this with squirrel weren't successful. Something about the order of applying the 'where' statements. I'll try again :)